### ISSUE TITLE
unable to see the whole text in the box. 

### ISSUE THREAD

**Issue by gauravshukla2203** [sagemath/website#439]
![image](https://user-images.githubusercontent.com/76643106/209564232-76152ad9-b0b3-4a96-8747-4c9ca46e7ba6.png)

**Comment by Jyoticharan** [comment]
I would be glad to work on this particular issue so please assign me the issue

**Comment by haraldschilly** [comment]
I don't think it makes any sense to work on that as it is stated. What's your plan to solve this, @Jyoticharan ? I would rather like to see a dynamically updated status of all mirrors, which does parse the contents of these sentinel files. However, unless https://github.com/sagemath/website/issues/459 is resolved, that information is outdated anyways
